qiita_db/processing_job.py

```diff
@@ -281,20 +336,28 @@ def heartbeat(self):
             qdb.sql_connection.TRN.add(sql, [self.id])
             return qdb.sql_connection.TRN.execute_fetchlast()
 
-    @heartbeat.setter
-    def heartbeat(self, value):
-        """Sets the timestamp for the last received heartbeat
+    def execute_heartbeat(self):
```

Comment:
This only changes the heartbeat from one state to another, right? If so, this name is confusing and the first line of the docstring is misleading.